src/zope/server/ftp/server.py

```diff
@@ -194,7 +194,7 @@ def cmd_dele(self, args):
     def cmd_help(self, args):
         'See IFTPCommandHandler'
         self.reply('HELP_START', flush=0)
```

Comment:
This is mostly an accident of implementation: I was trying to get the tests to pass, instead of stopping and thinking about what makes sense in the API. 

Thank you for paying attention to such details.  I'll think about this some more.

Reply:
Ok, so `LineServerChannel.reply()` doesn't take a reply *string*, it takes a reply *code*, which is used to look up the reply string in the `status_messages` dict.

TBH I initially assumed those codes would be integers (because why would I familiarize myself with code I'm trying to modify? there are tests so I wouldn't have to think /s).

Now about the encoding -- perhaps it *would* be cleaner if the status_messages dict contained bytestrings as values, and if the args passed to `reply()` also were bytes.  I'll think about this some more.

Reply:
Ok, given that I want Python 3.4 support after all, it'd be difficult to have reply implement %-formatting I require args to be bytes.  I'm going to keep this as is.

Besides, I don't expect zope.server FTP support to have many users.